LongReadRNABenchmark/LongReadRNABenchmarkTasks.wdl

```diff
@@ -0,0 +1,697 @@
+        String datasetName
+        Int numThreads
+    }
+
+    String docker = "us.gcr.io/broad-dsde-methods/kockan/isoseq3:latest"
+    Int cpu = 16
+    Int memory = 256
+    Int diskSizeGB = 500
+    File monitoringScript = "gs://broad-dsde-methods-tbrookin/cromwell_monitoring_script2.sh"
+
+    command <<<
+        bash ~{monitoringScript} > monitoring.log &
+
+        samtools fastq ~{inputBAM} > temp.fastq
+
+        pbmm2 align --num-threads ~{numThreads} --preset ISOSEQ --sort ~{referenceGenome} temp.fastq  pbmm_realigned.bam
+
+        isoseq3 collapse pbmm_realigned.bam "IsoSeq_out_~{datasetName}.gff"
+    >>>
+
+    output {
+        File isoSeqGFF = "IsoSeq_out_~{datasetName}.gff"
+        File monitoringLog = "monitoring.log"
+    }
+
+    runtime {
+        docker: docker
+        cpu: cpu
+        memory: "~{memory} GiB"
+        disks: "local-disk ~{diskSizeGB} HDD"
+    }
+}
+
+task Tama {
+    input {
+        File inputBAM
+        File inputBAMIndex
+        File referenceGenome
+        File referenceGenomeIndex
+        String datasetName
+    }
+
+    String docker = "us.gcr.io/broad-dsde-methods/kockan/tama:latest"
+    Int cpu = 16
+    Int memory = 256
+    Int diskSizeGB = 500
+    File monitoringScript = "gs://broad-dsde-methods-tbrookin/cromwell_monitoring_script2.sh"
+
+    String outputPrefix = "TAMA_out_~{datasetName}"
+
+    command <<<
+        bash ~{monitoringScript} > monitoring.log &
+
+        python tama_collapse.py -b BAM -s ~{inputBAM} -f ~{referenceGenome} -p ~{outputPrefix} -x capped
+    >>>
+
+    output {
+        File tamaBED = "~{outputPrefix}.bed"
+        File monitoringLog = "monitoring.log"
+    }
+
+    runtime {
+        docker: docker
+        cpu: cpu
+        memory: "~{memory} GiB"
+        disks: "local-disk ~{diskSizeGB} HDD"
+    }
+}
+
+task Flames {
+    input {
+        File inputBAM
+        File inputBAMIndex
+        File referenceGenome
+        File referenceGenomeIndex
+        File referenceAnnotation
+        String datasetName
+    }
+
+    String docker = "us.gcr.io/broad-dsde-methods/kockan/flames:latest"
+    Int cpu = 16
+    Int memory = 256
+    Int diskSizeGB = 500
+    File monitoringScript = "gs://broad-dsde-methods-tbrookin/cromwell_monitoring_script2.sh"
+
+    command <<<
+        bash ~{monitoringScript} > monitoring.log &
+
+        /usr/local/src/FLAMES/python/bulk_long_pipeline.py \
+        --gff3 ~{referenceAnnotation} \
+        --genomefa ~{referenceGenome} \
+        --outdir . \
+        --inbam ~{inputBAM}
+
+        mv isoform_annotated.gff3 FLAMES_out_~{datasetName}.gff
+    >>>
+
+    output {
+        File flamesGFF = "FLAMES_out_~{datasetName}.gff"
+        File monitoringLog = "monitoring.log"
+    }
+
+    runtime {
+        docker: docker
+        cpu: cpu
+        memory: "~{memory} GiB"
+        disks: "local-disk ~{diskSizeGB} HDD"
+    }
+}
+
+task ReducedAnnotationGFFCompare {
+    input {
+        File reducedAnnotationDB
```

Comment:
this should be a task that takes a reducedAnnotationDB and a single GTF, and does the comparison of that single GTF to the annotation DB.  Then in your workflow, you can call it separately on the output of each tool you are testing, instead of writing a separate comparison call for each tool into the task.